packages/www/layouts/streamDetail.tsx

```diff
@@ -458,6 +461,17 @@ const StreamDetail = ({
                       <Cell css={{ cursor: "pointer" }}>
                         <ShowURL url={globalIngestUrl} anchor={false} />
                       </Cell>
+                      <Cell css={{ color: "$mauve11" }}>SRT ingest URL</Cell>
+                      <Cell css={{ cursor: "pointer" }}>
+                        <ShowURL
+                          url={globalSrtIngestUrl}
+                          shortendUrl={globalSrtIngestUrl.replace(
+                            globalSrtIngestUrl.slice(38),
```

Comment:
Is this the `streamKey` length? WDYT of?:
```suggestion
                            globalSrtIngestUrl.slice(stream?.streamKey ?? 0),
```